local-volume/provisioner/cmd/main.go

```diff
@@ -39,17 +40,29 @@ import (
 )
 
 var (
-	optListenAddress string
-	optMetricsPath   string
+	optListenAddress  string
+	optMetricsPath    string
+	optDeleteContents string
 )
 
 func main() {
 	rand.Seed(time.Now().UTC().UnixNano())
 	flag.StringVar(&optListenAddress, "listen-address", ":8080", "address on which to expose metrics")
 	flag.StringVar(&optMetricsPath, "metrics-path", "/metrics", "path under which to expose metrics")
+	flag.StringVar(&optDeleteContents, "delete-contents", "", "delete contents in directory")
```

Comment:
It is kind of confusing to reuse the same process to do a different function.  Can we create a new util binary for this?  Or can the job just use a shell command to do the deletion?

Reply:
Updated to use shell script, see https://github.com/kubernetes-incubator/external-storage/pull/863/files#diff-ba78747a8f5a1366cfa9a5c8bb9b77a2.